homework/CSharpException/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CSharpException;
+
+Triangle triangle = new Triangle();
+try
+{
```

Comment:
You initialize properties and then pass them to the method. For what reason have you used both properties and variables? Better to use one of them. Initialize properties in the Program class and then use them in the class (delete variables from method) or use only variables (create 3 variables in the Program class, initialize them (int oneLength = int.Parse(Console.ReadLine())) and then pass them to the method.

Reply:
And the third option pass variables to the constructor, initialize properties in constructor.